ios/xcode.rb

```diff
@@ -20,3 +22,25 @@ def override_build_settings!(build_settings, overrides)
     build_settings[setting] = value
   end
 end
+
+def configure_xcschemes!(xcschemes_path, project_root, target_platform, name)
+  xcscheme = File.join(xcschemes_path, 'ReactTestApp.xcscheme')
+  metal_api_validation = platform_config('metalAPIValidation', project_root, target_platform)
+
+  # Oddly enough, to disable Metal API validation, we need to add `enableGPUValidationMode = "1"`
+  # to the xcscheme Launch Action.
+  if metal_api_validation == false
+    xcscheme_content = File.read(xcscheme)
+    doc = REXML::Document.new(xcscheme_content)
+    doc.root.elements['LaunchAction'].attributes['enableGPUValidationMode'] = '1'
+
+    File.open(xcscheme, 'w') do |file|
+      doc.write(file)
```

Comment:
Let's add indentation and make it more legible:
```suggestion
      doc.write(file, 3)
```
(Using 3 spaces because that's what Xcode generates…)